third_party/txt/src/minikin/MinikinInternal.cpp

```diff
@@ -24,7 +24,7 @@
 
 namespace minikin {
 
-std::recursive_mutex gMinikinLock;
+[[clang::no_destroy]] std::recursive_mutex gMinikinLock;
```

Comment:
This would need to be gated by an `#ifdef __clang__` directive

Reply:
Done!